cmd/system-probe/modules/traceroute.go

```diff
@@ -47,10 +49,11 @@ func (t *traceroute) Register(httpMux *module.Router) error {
 		start := time.Now()
 		vars := mux.Vars(req)
 		id := getClientID(req)
-		host := vars["host"]
-
-		cfg := tracerouteutil.Config{
-			DestHostname: host,
+		cfg, err := parseParams(vars)
+		if err != nil {
+			log.Errorf("invalid params: %s", err)
+			w.WriteHeader(http.StatusBadRequest)
```

Comment:
🥜 : write the invalid params are part of the HTTP response as well so the error message can be passed back to the regular agent for logging

Reply:
I just added the host in the logs, but for the vars I think it's fine to not have them since the `parseParams()` already returns an explicit message.